pipelines/_ingredients/ApPipeWithFakes.yaml

```diff
@@ -110,6 +111,10 @@ subsets:
       - transformDiaSrcCat
       - diaPipe
       - fakesMatch
+  qaMetrics:
+    subset:
+      - analyzeAssocDiaSrcCore
+      - diffimTaskCore
     description: >
       The AP pipeline with fakes. Requires apPipe and prepareFakes subsets.
```

Comment:
```suggestion
    description: >
      The AP pipeline with fakes. Requires apPipe and prepareFakes subsets.
  qaMetrics:
    subset:
      - analyzeAssocDiaSrcCore
      - diffimTaskCore
```

Reply:
The fakes definition of `apPipe` seems to be broken, and to have been previously worked around in the specialized `ApVerifyWithFakes` by using `processCcd` in place of `apPipe`. Not sure if you'd want to fix that, or have me do it, but it would probably be easiest to address it _before_ adding the QA metrics.

Reply:
I removed the `qaMetrics` subset and put the metrics in `apPipe`. That seems to fix the import problems you pointed out elsewhere. I have verified that a local `ap_verify` run of `ApPipeWithFakes.yaml` on the `ap_verify_ci_hits2015` works.

Reply:
Ok. Just to be clear, I meant for the new tasks to be added to `apPipe` _in addition to_ `qaMetrics`, not instead of. A `qaMetrics` subset might be useful (we don't have one in `ap_verify` because the way metrics are split among different files makes defining it impractical).

Reply:
Filed [DM-43349](https://rubinobs.atlassian.net/browse/DM-43349) to deal with the preexisting bad `apPipe` definition.